assets/stylesheets/_breakpoints.scss

```diff
@@ -10,6 +10,7 @@ $break-large: 960px;	// admin sidebar auto folds
 $break-medium: 782px;	// adminbar goes big
 $break-small: 600px;
 $break-mobile: 480px;
+$break-zoomedin: 280px;
```

Comment:
This variable name reads a little awkwardly to me — should it be`break-min-zoom` or something? Or perhaps we just need to add a dash in, so that it reads as `break-zoomed-in`. Interested in what @jasmussen thinks.

Reply:
Great thoughts. Both your suggestions are nice.

Reply:
Done! Going with `zoomed-in` so it's obvious what the use case scenario is.